python/desc/skycatalogs/skyCatalogs.py

```diff
@@ -196,6 +204,14 @@ def raw_config(self):
         '''
         return self._config
 
+    @property
```

Comment:
I just noticed that `observed_sed_factory` and `extinguisher` are already defined above.

Reply:
Fixed in next push.